action.yml

```diff
@@ -11,17 +11,19 @@ inputs:
   secrets:
     description: "One or more secret Ids to retrieve and the corresponding GitHub environment variable name to set"
     required: true
+  cloud_region:
+    description: "(Optional) The Bitwarden server region to use if cloud-hosted service is used. Either 'us' or 'eu'"
+    required: false
+    default: "us"
   base_url:
     description: "(Optional) For self-hosted bitwarden instances provide your https://your.domain.com"
     required: false
   identity_url:
     description: "(Optional) For self-hosted bitwarden instances provide your https://your.domain.com/identity"
     required: false
-    default: "https://identity.bitwarden.com"
```

Comment:
please update the README.md, which specifies the defaults for `identity_url` and `api_url`